examples/src/12-exchange.ts

```diff
@@ -9,8 +9,8 @@ async function main(): Promise<void> {
     networkName: NetworkNames.Mainnet,
   });
 
-  const exchangeSupportedAssets = await sdk.getExchangeSupportedAssets();
-  logger.log('found exchange supported assets', exchangeSupportedAssets.length);
+  const exchangeSupportedAssets = await sdk.getExchangeSupportedAssets({ page: 1, limit: 100 });
```

Comment:
you send an object = one variable
receiving end waits for two variables (page, limit)
so either need to pass two variables here 
or change the function itself, and other places where it is called

Reply:
oh wait, it's calling the dto method, nevermind